tests/foreman/ui/test_myaccount.py

```diff
@@ -16,96 +16,205 @@
 
-        @Steps:
-        1. Update current User with all variations of Email Address in [1]
+        @Steps: Update current User with all variations of Surname in [1]
 
         @expectedresults: Current User is updated
 
         @caseautomation: notautomated
+
+        @CaseImportance: Critical
         """
+        valid_strs = _valid_string_data()
+        valid_strs.append('name with space')
+        self.assert_text_field_update('users.lastname', *valid_strs)
 
-    @stubbed()
     @tier1
     def test_positive_update_language(self):
         """Update Language in My Account
 
         @id: 87604475-3a8e-4cb1-ace4-ea874b1d9e72
 
-        @Steps:
-        1. Update current User with all different Language options
+        @Steps: Update current User with all different Language options
 
         @expectedresults: Current User is updated
 
-        @caseautomation: notautomated
+        @CaseImportance: Critical
         """
 
-    @stubbed()
+        for lang, locale in LANGUAGES.items():
+            with self.subTest(lang):
+                user = User(login=gen_alpha(), password='password')
+                user.id = user.create().id
+                with Session(self.browser, user.login, user.password):
+                    self.my_account.navigate_to_entity()
+                    self.my_account.select(
+                        locators['users.language_dropdown'],
+                        lang
+                    )
+                    self.my_account.click(common_locators['submit'])
+                    self.my_account.wait_until_element_is_not_visible(
+                        common_locators["notif.success"])
+                    self.my_account.navigate_to_entity()
+                    option = self.my_account.wait_until_element_exists(
+                        locators['users.selected_lang'])
+                    # Cant use directly lang because its value changes
+                    # after updating current language
+                    self.assertEqual(
+                        locale,
+                        option.get_attribute('value')
+                    )
+                user.delete()
+
     @tier1
     def test_positive_update_password(self):
         """Update Password/Verify fields in My Account
 
         @id: 3ab5d347-e02a-4d34-aec0-970419525268
 
-        @Steps:
-        1. Update Password/Verify fields with all variations in [1]
+        @Steps: Update Password/Verify fields with all variations in [1]
 
         @expectedresults: User is updated
 
-        @caseautomation: notautomated
+        @CaseImportance: Critical
         """
+        for password in _valid_string_data(max_len=254):
+            with self.subTest(password):
+                with self.logged_test_user():
+                    self.my_account.navigate_to_entity()
+                    self.my_account.wait_until_element(
+                        locators['users.password'])
+                    self.my_account.assign_value(
```

Comment:
why don't we create an usual `update` helper like we did for any other entity (e.g. 'User')?